cli/src/main/scala-2.12/coursier/cli/options/CacheOptions.scala

```diff
@@ -5,7 +5,6 @@ import coursier.Cache
 
 final case class CacheOptions(
   @Help("Cache directory (defaults to environment variable COURSIER_CACHE or ~/.coursier/cache/v1)")
-  @Short("C")
```

Comment:
Removing this because conflicting with `--classifier / -C`. I think the annotation processor should throw an error when there are conflicts.

Reply:
I agree. A helper method could be added for that in case-app.